lib/main.dart

```diff
@@ -1,14 +1,21 @@
+import 'package:ecellapp/core/utils/logger.dart';
+import 'package:ecellapp/screens/signup/signup.dart';
+import 'package:ecellapp/screens/signup/signup_repository.dart';
 import 'package:flutter/material.dart';
 import 'package:flutter_bloc/flutter_bloc.dart';
 
 import 'core/utils/injection.dart';
-import 'screens/login/cubit/login_cubit.dart';
-import 'screens/login/login.dart';
-import 'screens/login/login_repository.dart';
+import 'screens/signup/cubit/signup_cubit.dart';
 
-Future<void> main() async {
+void main() async {
   WidgetsFlutterBinding.ensureInitialized();
   await init();
+
+  Log.i(
```

Comment:
🟤 
This is not necessarily required. remove it.

Reply:
made the change